apps/ledger-live-desktop/tests/page/portfolio.page.ts

```diff
@@ -81,9 +84,24 @@ export class PortfolioPage extends AppPage {
     await assetRowLocator.click();
   }
 
+  @step("Scroll to operations")
   async scrollToOperations() {
     await this.page.waitForTimeout(500);
     const operationList = this.page.locator("id=operation-list");
```

Comment:
might make sense to use [this locator](https://github.com/LedgerHQ/ledger-live/pull/8168/files#diff-2ebb39b2b4d0534aaa0564f39f516ec7b8762351e4f5db726d952331bb1f820aR12)